src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,217 @@
+        { argumentType: ArgumentTypes.STRING },
+      ],
+    },
+    'HF.DIVIDE': {
+      method: 'divide',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.EQ': {
+      method: 'eq',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.GT': {
+      method: 'gt',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.GTE': {
+      method: 'gte',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.LT': {
+      method: 'lt',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.LTE': {
+      method: 'lte',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.MINUS': {
+      method: 'minus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.MULTIPLY': {
+      method: 'multiply',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.NE': {
+      method: 'ne',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.POW': {
+      method: 'pow',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UMINUS': {
+      method: 'uminus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UNARY_PERCENT': {
+      method: 'unary_percent',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UPLUS': {
+      method: 'uplus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+  }
+
+  public add(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.ADD'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.addWithEpsilon(a, b)
+      )
+  }
+
+  public concat(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.CONCAT'),
+      (a: string, b: string) => a.concat(b)
+    )
+  }
+
+  public divide(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.DIVIDE'),
+      divide
```

Comment:
Why `divide` is exported differently from the arithmetic helper? We can `this.interpreter.arithmeticHelper.subtract` but not `this.interpreter.arithmeticHelper.divide` 🙄 We could import all the functions from arithmetic helper and would be correct as well, but can we use one, consistent manner?

Reply:
Only functions that need context (e.g. config) are part of 'object' in ArithmeticHelper, while context-free are only part of module.